build.js

```diff
@@ -149,7 +149,7 @@ function build(runtime, version, abi) {
   if (process.platform === 'win32') {
     if (version.split('.')[0] >= 4) {
       process.env.msvs_toolset = 15
-      process.env.msvs_version = 2017
+      process.env.msvs_version = argv.msvs_version || 2017
```

Comment:
What's the reason for this?

Reply:
Appveyor provided VS 2017 while GitHub Actions only provides VS 2019 by default. The build script had 2017 hardcoded so there was no way of changing it (which would normally affect users, but alas, users can't build this project at install time anyway, so that is a problem for later). Since the projects build correctly on 2019 as well, I didn't see the need to do a custom setup of 2017, and instead allowed passing in the version as an argument. I'm happy to hardcode 2019 instead, but I didn't want to change the default non-overloaded value in case there was an additional need for 2017 to be the default that I did not know about.

Reply:
Alright. Merging.